### PR TITLE
Portuguese Translation

### PR DESCRIPTION
**Pull request by DaniloOliveira28**
I translated the application to portuguese.

### CODE REVIEW

File: ui/src/main/resources/FileManager/FileManagerTranslations.pt_BR.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  <template/>
+  <defaultTemplate/>
+  <validationScript/>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+  <object>
+    <class>
+      <name>XWiki.TranslationDocumentClass</name>
```
Comment (mflorea) [inline ui/src/main/resources/FileManager/FileManagerTranslations.pt_BR.xml]: This object doesn't belong here. It should be already present on the default translation.

File: ui/src/main/resources/FileManager/FileManagerTranslations.pt_BR.xml [hunk 2]
```diff
@@ -0,0 +1,90 @@
+<xwikidoc>
+  <web>FileManager</web>
+  <name>FileManagerTranslations</name>
+  <language/>
+  <defaultLanguage/>
```
Comment (mflorea) [inline ui/src/main/resources/FileManager/FileManagerTranslations.pt_BR.xml]: The language and default language fields are not right. The language should be pt_BR and the default language should be en. I'm not sure how you generated this document.

File: ui/src/main/resources/FileManager/Translations.pt_BR.xml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<xwikidoc>
+  <web>FileManager</web>
+  <name>Translation</name>
+  <language>fr</language>
+  <defaultLanguage/>
```
Comment (mflorea) [inline ui/src/main/resources/FileManager/Translations.pt_BR.xml]: Looks like you edited the French translation. The language and default language are not good.

**Comment by mflorea** [comment]
Hi @DaniloOliveira28,

Thanks for your pull request. There are some issues that can be fixed but what is more important is that these translation files are used by the old version of the File Manager application (v1.x). The new version (v2.x) uses a different translation document (``FileManagerCode.Translations``). It would be awesome if you can translate the new keys in Brazilian Portuguese using the [localization tool](http://l10n.xwiki.org/xwiki/bin/view/Contrib/FileManager?language=pt_BR). You will need an account. Thanks.

**Comment by DaniloOliveira28** [comment]
Hello @mflorea 

You are welcome. In the next pull requests I will pay attention to your annonnations.

Danilo

**Pull request closed without merging**